src/Common/nunit/CommandLineOptions.cs

```diff
@@ -296,6 +298,23 @@ protected virtual void ConfigureOptions()
             this.Add("where=", "Test selection {EXPRESSION} indicating what tests will be run. See description below.",
                 v => WhereClause = RequiredValue(v, "--where"));
 
+            this.Add("params|D=", "Define a test parameter.",
```

Comment:
`--params` and `-D` seem a bit confusing especially since we aren't using `-p` or `-P`.